src/App.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { TextField } from './TextField';
+
+// props
+// hooks
+// render props
+
+const App: React.FC = () => {
+  return <div>
+    <TextField text='hello' person={{firstName: '', lastName: ''}}/>
+  </div>
```

Comment:
I don't think I ended up going back to this yet. Will need to update. 
Also might be a good idea to return ()